### ISSUE TITLE
[feature request]: polygons overlapping on the antimeridian

### ISSUE THREAD

**Issue by kkdd** [mclaeysb/geojson-polygon-self-intersections#1]
I would request geojson-polygon-self-intersections would treat polygons crossing on the antimeridian correctly.
ps. [rbush](https://github.com/mourner/rbush) doesn't presently cope with the antimeridian.
ps2. I have extended [typescript-javascript-hilbert-r-tree](https://github.com/jorgenkg/typescript-javascript-hilbert-r-tree) to [javascript-hilbert-r-tree](https://github.com/kkdd/javascript-hilbert-r-tree) with treatment of the antimeridian.

**Comment by mclaeysb** [comment]
Good point. Can you specify what you mean with 'treating correctly'? Don't hesitate to submit a PR if you have the time to make one!
